package.json

```diff
@@ -93,8 +93,8 @@
     "eslint-plugin-react-hooks": "^4.6.2",
     "eslint-plugin-testing-library": "^6.2.2",
     "husky": "^8.0.3",
-    "i18next": "^23.16.4",
-    "i18next-parser": "^9.0.2",
+    "i18next": "21.10.0",
+    "i18next-parser": "8.13.0",
```

Comment:
i18n versions are a bit finicky, these versions currently give this error if you run the extract-translations command. 
<img width="1022" alt="Screenshot 2024-12-05 at 9 55 48 PM" src="https://github.com/user-attachments/assets/dd8a642d-32c6-4956-955e-71df38bd3cab">

Reply:
Pushed a commit that updates the parser version

Reply:
@samuelmale reverting it would give this issue

Reply:
Ohh yes, I had missed that! How about updating the parser version across the framework?